src/EnergyPlus/DataSizing.hh

```diff
@@ -451,14 +451,17 @@ namespace DataSizing {
 		bool EMSOverrideDesCoolMassOn; // true if EMS is acting on this structure
 		Real64 EMSValueDesCoolMassFlow; // Value EMS directing to use for Design Cooling air mass flow [kg/s]
 		Real64 DesHeatLoad; // zone design heating load [W]
+		Real64 NonAirSysDesHeatLoad; // same as zone design heating load without special multipliers from Sizing:Zone object [W]
```

Comment:
I don't think this is the correct description.  This *should* include any sizing factor fro sizing:zone and/or sizing:parameters.  What it doesn't include is any scaling dues to airflow rates which could come from user-specified flow rates in sizing:zone or sizing:system, and/or sizing on ventilation requirement.  Suggest:
```
Real64 DesHeatLoad; // zone design heating load including sizing factor and scaled to match airflow sizing
Real64 NonAirSysDesHeatLoad; // base zone design heating load including sizing factor
```
Similar for cooling.